conda/common/logic.py

```diff
@@ -920,8 +943,8 @@ def sum_val(sol, odict):
                 log.trace('Beginning sum minimization')
                 objval = sum_val
 
-            odict = {a: c for c, a in objective}
-            bestval = objval(bestsol, odict)
+            objective_dict = {a: c for c, a in zip(coeffs, lits)}
```

Comment:
Renamed this to `objective_dict` since `odict` is used for `OrderedDict` elsewhere in the code base.